src/sentry/static/sentry/app/utils/getDynamicText.tsx

```diff
@@ -2,12 +2,12 @@
 
 // Return a specified "fixed" string when we are in a testing environment
 // (more specifically in a PERCY env (e.g. CI))
-export default function getDynamicText({
+export default function getDynamicText<T>({
```

Comment:
In another pull request you have this changing to a different generic signature. Is the other implementation the correct one, or is it this one?

Reply:
The other implementation is the correct one. This changeset is split out into a new PR: https://github.com/getsentry/sentry/pull/14415